lib/core/public/cleanup-plugins.js

```diff
@@ -42,4 +42,8 @@ function cleanupPlugins(resolve, reject) {
   })
   .catch(reject);
 }
+
+function cleanupPlugins(resolve=(() => {}), reject=axe.log) {

```

Comment:
You don't need to define a new function for this. Please just put these default params into the original function.

Reply:
I had to do this because of:

```
Warning: lib/core/public/cleanup-plugins.js: Non-simple parameter in strict mode (2:24) Use --force to continue.
```

In order to put default params in the original function signature, I would have to disable strict mode, which I don't think you want to do.

Reply:
This seems to be a JSHint bug. Just use fallback params instead:
```js
resolve = resolve || () => {};
reject = reject || axe.log;
```